news/_posts/2020-08-29-Polymake.jl-0.5-release.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: post
+title: Polymake.jl-0.5 release
+author: Marek Kaluba
+---
+
+`Polymake.jl` v0.5.0 has been released!
+
+The main and show-stealing reason to bump the major version is the transition of `Polymake.jl` from `BinaryBuilder` dependencies to modern Julia artifacts system. Thanks to the transition there is literally no compilation happening during `add Polymake` and all our libraries should be shining and working with each other.
+
+A huge round of applause is due to @benlorenz, who single-handedly cross-compiled `perl` for MacOsX on a MIPS router dev-board to make this happen. Without him, nobody would dare to look into the eyes of the beast.
+
+Happy coding,
```

Comment:
Like for the 0.4 release, newlines are not rendered properly:

![new-poly-0 5](https://user-images.githubusercontent.com/8462914/93187116-b9c64300-f73f-11ea-9702-8310675dda5d.png)

Cf https://github.com/oscar-system/oscar-website/pull/123/files#diff-7e42fc0bb0b1c85e278471542cb1f5f4R17, were you ended up using `<br/>` and `_Note_`.

I pushed this branch to https://rfourquet.github.io/oscar-website/news/2020/08/29/Polymake.jl-0.5-release/ if you like to have a look at the whole post.

Reply:
thanks, somehow  `\n` are not respected by the website generator, I tried to add double linebreaks and lets see how it goes;
If not will add `<br/>`. Could you push to your repo?

Reply:
I pushed, double line breaks don't seem to work better.